src/porepy/models/constitutive_laws.py

```diff
@@ -4174,13 +4249,17 @@ def elastic_normal_fracture_deformation(
 
         # Normal stiffness (as per Barton-Bandis terminology). Units: Pa / m
         normal_stiffness = self.fracture_normal_stiffness(subdomains)
+        # Rescale, since contact traction is dimensionless. TODO: Decide if this should
```

Comment:
Ah, yes. This mirrors my comment above. I think the natural choice would be to hide it from a method like this. Otherwise, one needs to be aware of what quantity already has been nondimensionalized or not.

Reply:
Shall we discuss this one briefly in person?